bigtable/beam/workload-generator/README.md

```diff
@@ -96,5 +105,5 @@ If you would like to modify this and run it yourself you can use these commands:
    "-Dexec.args=--bigtableInstanceId=$INSTANCE_ID =--bigtableTableId=$TABLE_ID \
    --runner=dataflow --project=$GOOGLE_CLOUD_PROJECT \
    --region=$REGION" \
-   --workloadQPS=$WORKLOAD_QPS 
+   --workloadRate=$WORKLOAD_QPS 
```

Comment:
should we change the env variable name too?

Reply:
Done